openjdk9/src/test/resources/openjdk9-structure.yaml

```diff
@@ -0,0 +1,13 @@
+# Test spec for the Cloud Runtimes container structure testing framework.
+#
+# See https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/structure_tests/README.md for more detail.
+#
+schemaVersion: 1.0.0
+
+commandTests:
+- name: 'correct java version is installed'
+  command: ['java', '-version']
+  expectedError: ['openjdk version "${openjdk.version.major}-Debian"']
```

Comment:
This is interesting. They changed the -version output: https://blogs.oracle.com/java-platform-group/a-new-jdk-9-version-string-scheme and http://openjdk.java.net/jeps/223 -- we should follow this with our parsing logic so that it stays maintainable and we don't lose out on ensuring the minor version. 

Based on above I'm confused by the ``-Debian`` part - where did that get introduced. It also might be part of the major version?

```
openjdk version "9-Debian"
OpenJDK Runtime Environment (build 9-Debian+0-9b181-2)
OpenJDK 64-Bit Server VM (build 9-Debian+0-9b181-2, mixed mode)

```
java 8:
````
Start command: java -showversion -Xms51137M -Xmx51137M -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:+PrintCommandLineFlags -version
-XX:InitialHeapSize=53621030912 -XX:MaxHeapSize=53621030912 -XX:+ParallelRefProcEnabled -XX:+PrintCommandLineFlags -XX:+UseG1GC 
openjdk version "1.8.0_131"
OpenJDK Runtime Environment (build 1.8.0_131-8u131-b11-1~bpo8+1-b11)
OpenJDK 64-Bit Server VM (build 25.131-b11, mixed mode)
````